src/VisualStudio/Core/Def/Implementation/DesignerAttribute/DesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -61,15 +63,15 @@ internal partial class DesignerAttributeIncrementalAnalyzer : ForegroundThreadAf
         public Task DocumentResetAsync(Document document, CancellationToken cancellationToken)
         {
             _state.Remove(document.Id);
-            return _state.PersistAsync(document, new Data(VersionStamp.Default, VersionStamp.Default, designerAttributeArgument: null), cancellationToken);
+            return _state.PersistAsync(document, new Data(VersionStamp.Default, VersionStamp.Default, designerAttributeArgument: string.Empty), cancellationToken);
```

Comment:
📝 Consumers of `Data` seem to assume that `DesignerAttributeArgument` is not null. It seemed easiest to just change this construction site. `CanReusePersistedVersion.CanReusePersistedVersion` will eventually prevent this value from being used here:

https://github.com/dotnet/roslyn/blob/1477e5781b4043d6e569becfdd2b6fd46c10dce2/src/VisualStudio/Core/Def/Implementation/DesignerAttribute/DesignerAttributeIncrementalAnalyzer.cs#L88-L96